src/main/java/com/epam/reportportal/listeners/ListenerParameters.java

```diff
@@ -84,6 +85,7 @@ public ListenerParameters() {
 
 	public ListenerParameters(PropertiesLoader properties) {
 		this.description = properties.getProperty(DESCRIPTION);
+		this.apiKey = properties.getProperty(API_KEY, "");
```

Comment:
if you add api key, why uuid is still there? shouldn't we remove this property

Reply:
done